proto/internalapi/sensor/compliance_nodescan_iservice.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
```

Comment:
Why not just update `internalapi/sensor/compliance_iservice.proto`? It already is implemented to control the communication between Compliance pods and Sensor. Node scanning can just be added to the `oneof` in `MsgFromCompliance`

Reply:
Originally, I wanted to make sure we have completely separated processing pipelines for the two different NodeScan messages.
With a better idea of the work involved to set up a new bidi stream, your idea makes way more sense. Let me set that up!